GOGOT/app/src/main/java/com/example/gogot/ui/GameActivity.java

```diff
@@ -63,13 +71,23 @@ public void drawInitialBoard(BoardCard[][] boardCards, ArrayList<BoardCard> card
     }
 
     @Override
-    public void movePlayer(Point newPlayerPosition) {
+    public void removeIllumination(BoardCard[][] boardCards) {
+        gameBoard.removeIllumination(boardCards);
+    }
 
+    @Override
+    public void movePlayer(BoardCard playerCard, Point newPlayerPosition) {
```

Comment:
playerCard and newPlayerPosition are not required here both, it would be anough to  leave only one of them here